### PR TITLE
chore(Labs): Flattens labs file structure [GM-201]

### PR DESCRIPTION
**Pull request by codecaaron**
### Overview

Removes the distinction between brand / experimental to make this a blanket component library.

- Flattens labs to a single level structure `brand` and `experimental`.  
- Flattens other index files.

### PR Checklist

- [ ] Related to designs:
- [x] Related to JIRA ticket: GM-201
- [x] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.3.29-alpha.69de15.0<br/>@codecademy/gamut-labs@13.5.6-alpha.69de15.0<br/>@codecademy/styleguide@35.0.3-alpha.69de15.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.3.29-alpha.33a1aa.0<br/>@codecademy/gamut-labs@13.5.6-alpha.33a1aa.0<br/>@codecademy/styleguide@35.0.3-alpha.33a1aa.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://607c94bc20bfc4002a542e44--gamut-preview.netlify.app>https://607c94bc20bfc4002a542e44--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/607c94bc20bfc4002a542e44>Deploy Logs</a></p>

File: packages/gamut-labs/src/landingPage/Title.tsx [hunk 1]
```diff
@@ -24,7 +21,6 @@ export const Title: React.FC<TitleProps> = ({
   <Box maxWidth="58rem" className={className}>
     <StyledText
       as={isPageHeading ? 'h1' : 'h2'}
-      fontWeight="title"
```
Comment (JoshuaKGoldberg) [inline packages/gamut-labs/src/landingPage/Title.tsx:27]: Should this be in a separate PR?

**Review by JoshuaKGoldberg — CHANGES_REQUESTED** [review]
Unintended changes towards the end?

Also, maybe we should split labs into two packages?

File: packages/gamut-labs/src/landingPage/Title.tsx (hunk 1 above)
Comment (codecaaron) [reply packages/gamut-labs/src/landingPage/Title.tsx]: Heh yeah, undid any changes.  Guess I think I autosuggested my way into this :P

**Review by JoshuaKGoldberg — APPROVED** [review]
I think it makes sense to start the conversation soon about splitting packages, but until then this makes sense to me :+1:. Thanks!

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.3.29-alpha.514e9e.0<br/>@codecademy/gamut-labs@13.5.6-alpha.514e9e.0<br/>@codecademy/styleguide@35.0.3-alpha.514e9e.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://607d85868a19395ffa2841d0--gamut-preview.netlify.app>https://607d85868a19395ffa2841d0--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/607d85868a19395ffa2841d0>Deploy Logs</a></p>

**Pull request merged**